### PR TITLE
add class names to percentages that are over .5

### PR DESCRIPTION
**Pull request by Obleo33**

### CODE REVIEW

File: src/DistrictCards/DistrictYears.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+
   return (
     <div className="yearly-data">
-      {Object.keys(yearlyData.yearlyData).map((year, index) => {
-        return (
-          <div key={index}>{year} : {yearlyData.yearlyData[year]}</div>
-        )
+      {Object.keys(Data).map((year, index) => {
+        if(Data[year] >= .5){
+          return <div className="year over" key={index}>{year} : {Data[year]}</div>
```
Comment (Obleo33) [inline src/DistrictCards/DistrictYears.js:10]: if the data is over .5 we add the className of over

File: src/DistrictCards/DistrictYears.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 
 const DistrictYears = (yearlyData) => {
+  const Data = yearlyData.yearlyData;
```
Comment (Obleo33) [inline src/DistrictCards/DistrictYears.js:4]: cleaning up our data

File: src/DistrictCards/DistrictYears.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
     <div className="yearly-data">
-      {Object.keys(yearlyData.yearlyData).map((year, index) => {
-        return (
-          <div key={index}>{year} : {yearlyData.yearlyData[year]}</div>
-        )
+      {Object.keys(Data).map((year, index) => {
+        if(Data[year] >= .5){
+          return <div className="year over" key={index}>{year} : {Data[year]}</div>
+        }
+        return <div className="year under" key={index}>{year} : {Data[year]}</div>
```
Comment (Obleo33) [inline src/DistrictCards/DistrictYears.js:12]: data under .5 will have the className of under

File: src/DistrictCards/DistrictYears.js (hunk 1 above)
Comment (becs919) [reply src/DistrictCards/DistrictYears.js:10]: great! we can use for css later

File: src/DistrictCards/DistrictYears.js (hunk 2 above)
Comment (becs919) [reply src/DistrictCards/DistrictYears.js:4]: love clean up

File: src/DistrictCards/DistrictYears.js (hunk 3 above)
Comment (becs919) [reply src/DistrictCards/DistrictYears.js:12]: nice work

**Comment by becs919** [comment]
everything looks great

**Pull request merged**